libr/anal/p/anal_arm_cs.c

```diff
@@ -3288,6 +3271,29 @@ static int analop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len, RAn
 				analop64_esil (a, op, addr, buf, len, &handle, insn);
 			}
 		} else {
+			if (a->bits == 16) {
+				//patch analysis if insn in IT block
```

Comment:
It's a quite long (horizontally) construction, I think it could be extracted into a separate function.

Reply:
Done